README.md
# w3651t3


changelog.txt
@Version 	1.1.2
@Date		Sunday, September 20, 2015
@Author		Dorin Grigoras
	* New: Flot Charts (changes: added shortcode-charts.html, assets/plugins/chart.flot/)
	* Fix: Various small fixes (changes: assets/css/layout.css)


@Version 	1.1.1
@Date		Friday, September 18, 2015
@Author		Dorin Grigoras
	* New: Corporate Page 7 (changes: added index-corporate-7.html)
	* Update: Documentation - Revolution Slider 5 Local File
	* Fix: Admin left menu after browser resize (changes: admin/assets/css/layout.css, admin/assets/css/color_scheme/green.css)
	* Fix: Admin content resize (changes: admin/assets/js/app.js)
	* Fix: Revolution Slider v5 arrows bottom position (changes: assets/css/layout.css removed .arrow-middle, added .arrow-bottom - used on: rs-15.html, rs-16.html)
	

@Version 	1.1.0
@Date		Monday, September 14, 2015
@Author		Dorin Grigoras
	* New: Revolution Slider (RS) Plugin V5.0.8 (changes: added assets/plugins/slider.revolution.v5)
	* New: 32 Revolution Slider Examples (changes: start.html, added: assets/images/demo/rs5/)
	* New: RS Page - Carousel Highlight (added: rs-1.html)
	* New: RS Page - Carousel Instagram (added: rs-2.html)
	* New: RS Page - Carousel Media (added: rs-3.html)
	* New: RS Page - Carousel Classic(added: rs-4.html)
	* New: RS Page - Carousel Showcase (added: rs-5.html)
	* New: RS Page - Carousel Photography (added: rs-6.html)
	* New: RS Page - Rotating Words (added: rs-7.html)
	* New: RS Page - Big Bold (added: rs-8.html)
	* New: RS Page - Slider Fashion (added: rs-9.html)
	* New: RS Page - Slider Gym (added: rs-10.html)
	* New: RS Page - Slider Restaurant (added: rs-11.html)
	* New: RS Page - Slider Product Dark (added: rs-12.html)
	* New: RS Page - SliderProduct Light (added: rs-13.html)
	* New: RS Page - Slider Classic (added: rs-14.html)
	* New: RS Page - Gallery Light (added: rs-15.html)
	* New: RS Page - Gallery Dark (added: rs-16.html)
	* New: RS Page - Slider Fancy Text (added: rs-17.html)
	* New: RS Page - Slider Highlight (added: rs-18.html)
	* New: RS Page - Slider Media (added: rs-19.html)
	* New: RS Page - Slider News (added: rs-20.html)
	* New: RS Page - Slider Vimeo (added: rs-21.html)
	* New: RS Page - Slider Youtube (added: rs-22.html)
	* New: RS Page - Slider Travel (added: rs-23.html)
	* New: RS Page - Hero Image (added: rs-24.html)
	* New: RS Page - Hero News (added: rs-25.html)
	* New: RS Page - Hero Newsletter (added: rs-26.html)
	* New: RS Page - Hero Search (added: rs-27.html)
	* New: RS Page - Hero Sports (added: rs-28.html)
	* New: RS Page - Hero Vimeo (added: rs-29.html)
	* New: RS Page - Hero Youtube (added: rs-30.html)
	* New: RS Page - Premium Content Zoom (added: rs-31.html)
	* New: RS Page - Premium Food Carousel (added: rs-32.html)
	* New: Hover Buttons [shortcode-buttons-2.html] (changes: assets/css/color_scheme/*, assets/css/plugin-hover-buttons.css, assets/css/layout-dark.css)
	* Update: Fontawesome to V4.4.0 (changes: assets/css/essentials.css , assets/fonts/fontawesome-* - on admin too)

@Version 	1.0.2
@Date		Monday, September 07, 2015
@Author		Dorin Grigoras
	* New: About Us - Layout 5 (changes: added page-about-us-5.html)
	* New: Documentation more info (demo files, google fonts)
	* New: optional css file to rewite the default fonts(added: assets/css/_layout-font-rewrite.css)
	* New: Parallax enabled by default on mobile devices (changes: assets/css/essentials.css)
	* Fix: RTL Sliders/Carousels (changes: assets/css/layout-RTL.css)
	* Fix: header-side container width for < 1600px resolution (changes: assets/css/layout.css)



@Version 	1.0.1
@Date		Thursday, September 03, 2015
@Author		Dorin Grigoras
	* New: Documentation more info
	* New: admin fullcalendar language example (changes: admin/assets/js/view/demo.calendar.js)
	* Update: Layer Slider to 5.6.0 (changes: assets/plugins/slider.layerslider)
	* Fix: newsletter php code (changes: assets/php/newsletter.php - thanks to Cristea Mihai)
	* Fix: transparent header color on mobile (changes: assets/css/layout.css)
	* Fix: education template - mobile menu color (changes: assets/css/thematic-edication.css)
	* Fix: Various small fixes (changes: assets/css/layout.css , assets/css/layout-shop.css, assets/css/thematics-music.css, assets/css/layout-RTL.css)


@Version 	1.0.0
@Date		Tuesday, August 25, 2015
@Author		Dorin Grigoras
	* Initial Release

page-start.php
<!DOCTYPE html>
<!--[if IE 8]>			<html class="ie ie8"> <![endif]-->
<!--[if IE 9]>			<html class="ie ie9"> <![endif]-->
<!--[if gt IE 9]><!-->	<html> <!--<![endif]-->
<head>
    <meta charset="utf-8" />
    <title>CEITECHS - CUTTING EDGE INNOVATIVE TECHNOLOGIES FOR YOUR BUSINESS</title>
    <meta name="keywords" content="innovative,technology,cloud,software tanzania,systems Tanzania,IT Tanzania" />
    <meta name="description" content="Nobody knows tomorrow, but we can transform your business by impressing your clients and beating the competition with our smart innovative solutions." />
    <meta name="Author" content="Iddy Magohe [www.ceitechs.com]" />

    <!-- mobile settings -->
    <meta name="viewport" content="width=device-width, maximum-scale=1, initial-scale=1, user-scalable=0" />
    <!--[if IE]><meta http-equiv='X-UA-Compatible' content='IE=edge,chrome=1'><![endif]-->

    <!-- WEB FONTS : use %7C instead of | (pipe) -->
    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400%7CRaleway:300,400,500,600,700%7CLato:300,400,400italic,600,700" rel="stylesheet" type="text/css" />

    <!-- CORE CSS -->
    <link href="assets/plugins/bootstrap/css/bootstrap.min.css" rel="stylesheet" type="text/css" />

    <!-- THEME CSS -->
    <link href="assets/css/essentials.css" rel="stylesheet" type="text/css" />
    <link href="assets/css/layout.css" rel="stylesheet" type="text/css" />

    <!-- PAGE LEVEL SCRIPTS -->
    <link href="assets/css/header-1.css" rel="stylesheet" type="text/css" />
    <link href="assets/css/color_scheme/green.css" rel="stylesheet" type="text/css" id="color_scheme" />
</head>
	<!--
		AVAILABLE BODY CLASSES:
		
		smoothscroll 			= create a browser smooth scroll
		enable-animation		= enable WOW animations

		bg-grey					= grey background
		grain-grey				= grey grain background
		grain-blue				= blue grain background
		grain-green				= green grain background
		grain-blue				= blue grain background
		grain-orange			= orange grain background
		grain-yellow			= yellow grain background
		
		boxed 					= boxed layout
		pattern1 ... patern11	= pattern background
		menu-vertical-hide		= hidden, open on click
		
		BACKGROUND IMAGE [together with .boxed class]
		data-background="assets/images/boxed_background/1.jpg"
	-->
	<body class="smoothscroll enable-animation">
		<!-- wrapper -->
		<div id="wrapper">

			<!-- 
				AVAILABLE HEADER CLASSES

				Default nav height: 96px
				.header-md 		= 70px nav height
				.header-sm 		= 60px nav height

				.noborder 		= remove bottom border (only with transparent use)
				.transparent	= transparent header
				.translucent	= translucent header
				.sticky			= sticky header
				.static			= static header
				.dark			= dark header
				.bottom			= header on bottom
				
				shadow-before-1 = shadow 1 header top
				shadow-after-1 	= shadow 1 header bottom
				shadow-before-2 = shadow 2 header top
				shadow-after-2 	= shadow 2 header bottom
				shadow-before-3 = shadow 3 header top
				shadow-after-3 	= shadow 3 header bottom

				.clearfix		= required for mobile menu, do not remove!

				Example Usage:  class="clearfix sticky header-sm transparent noborder"
			-->
			<div id="header" class="sticky clearfix">

				<!-- TOP NAV -->
                <?php
                require_once("menu_header.html");
                ?>
				<!-- /Top Nav -->

			</div>


			<!-- 
				PAGE HEADER 
				
				CLASSES:
					.page-header-xs	= 20px margins
					.page-header-md	= 50px margins
					.page-header-lg	= 80px margins
					.page-header-xlg= 130px margins
					.dark			= dark page header

					.shadow-before-1 	= shadow 1 header top
					.shadow-after-1 	= shadow 1 header bottom
					.shadow-before-2 	= shadow 2 header top
					.shadow-after-2 	= shadow 2 header bottom
					.shadow-before-3 	= shadow 3 header top
					.shadow-after-3 	= shadow 3 header bottom
			-->
			<section class="page-header page-header-xs dark">
				<div class="container">

					<h1>GETTING STARTED HAS NEVER BEEN EASIER.</h1>

					<!-- breadcrumbs -->
					<ol class="breadcrumb">
						<li><a href="#">Home</a></li>
						<li><a href="#">Pages</a></li>
						<li class="active">Services</li>
					</ol><!-- /breadcrumbs -->

				</div>
			</section>
			<!-- /PAGE HEADER -->




			<!-- -->


				<div class="container">

					<div class="box-inner">
					
						<div class="timeline"><!-- .timeline-inverse = right position [left on RTL] -->

								<!-- ITEM -->
								<div class="timeline-item timeline-item-bordered">
									<!-- timeline entry -->
									<div class="timeline-entry rounded"><!-- .rounded = entry -->
										01<span>offer</span>
										<div class="timeline-vline"><!-- vertical line --></div>
									</div>
									<!-- /timeline entry -->

									<h2 class="uppercase bold size-17">We get to know you, your business challenges, and your goals.</h2>
									Do you need a new system, or a website or maybe a mobile app? Do you need to organize big data? Do you think you should be getting more out of your existing systems? Or do you think that you’re missing something?.</p>
                                    <header class="text-center margin-bottom-30">
                                        <h2 class="weight-500 letter-spacing-1 size-13"><span>We conduct detailed interviews and workshops with key personnel from your company to learn about your current and future needs.</span></h2>
                                    </header>



								</div>
								<!-- /ITEM -->

								<!-- ITEM -->
								<div class="timeline-item timeline-item-bordered">
									<!-- timeline entry -->
									<div class="timeline-entry rounded"><!-- .rounded = entry -->
										02<span>offer</span>
										<div class="timeline-vline"><!-- vertical line --></div>
									</div>
									<!-- /timeline entry -->

									<h2 class="uppercase bold size-17">We identify opportunities for improvement.</h2>
									<p>If your newest software is speaking the equivalent of Chinise, but your legacy system only speaks Hindi, do you have to toss out both and start over? Not necessarily. Maybe you just need to insert a good translator between them. </p>
                                    <header class="text-center margin-bottom-30">
                                        <h2 class="weight-500 letter-spacing-1 size-13"><span>We will find the best fit Integration Pattern for your needs, There are many ways to get your systems to work together in harmony. </span></h2>
                                    </header>

								</div>
								<!-- /ITEM -->

								<!-- ITEM -->
								<div class="timeline-item timeline-item-bordered">
									<!-- timeline entry -->
									<div class="timeline-entry rounded"><!-- .rounded = entry -->
										03<span>offer</span>
										<div class="timeline-vline"><!-- vertical line --></div>
									</div>
									<!-- /timeline entry -->

									<h2 class="uppercase bold size-17">We help you craft your requirements.</h2>
									<p>Description of a software system or process to be developed, laying out functional and non-functional requirements, and may include a set of use cases that describe interactions the users will have with the software.</p>
                                    <header class="text-center margin-bottom-30">
                                        <h2 class="weight-500 letter-spacing-1 size-13">
                                            <span>We will help you baseline Acceptance Tests, External APIs and Infrastructure you will interact with &amp; Performance Requirements</span>
                                        </h2>
                                    </header>

								</div>
								<!-- /ITEM -->

								<!-- ITEM -->
								<div class="timeline-item timeline-item-bordered">
									<!-- timeline entry -->
									<div class="timeline-entry rounded"><!-- .rounded = entry -->
										04<span>offer</span>
										<div class="timeline-vline"><!-- vertical line --></div>
									</div>
									<!-- /timeline entry -->

									<h2 class="uppercase bold size-17">We provide detailed, honest estimates we can stand by.</h2>
									<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas metus nulla, commodo a sodales sed, dignissim pretium nunc. Nam et lacus neque. Ut enim massa, sodales tempor convallis et, iaculis ac massa. Maecenas metus nulla, commodo a sodales sed, dignissim pretium nunc. Nam et lacus neque.</p>

									<div class="row">
										<div class="col-md-4 col-sm-4 col-xs-6">
											<img class="img-responsive" src="assets/images/demo/wb/WB0271X52-min.png" alt="image" />
										</div>

										<div class="col-md-4 col-sm-4 col-xs-6">
											<img class="img-responsive" src="assets/images/demo/wb/WB0PNJ9B6-min.png" alt="image" />
										</div>

										<div class="col-md-4 col-sm-4 col-xs-12">
											<img class="img-responsive" src="assets/images/demo/wb/WB05SR527-min.png" alt="image" />
										</div>
									</div>
								</div>
								<!-- /ITEM -->

                            <!-- ITEM -->
                            <div class="timeline-item timeline-item-bordered">
                                <!-- timeline entry -->
                                <div class="timeline-entry rounded"><!-- .rounded = entry -->
                                    05<span>offer</span>
                                    <div class="timeline-vline"><!-- vertical line --></div>
                                </div>
                                <!-- /timeline entry -->

                                <h2 class="uppercase bold size-17">We get to work — balancing efficiency, quality, and your timelines.</h2>
                                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas metus nulla, commodo a sodales sed, dignissim pretium nunc. Nam et lacus neque. Ut enim massa, sodales tempor convallis et, iaculis ac massa. Maecenas metus nulla, commodo a sodales sed, dignissim pretium nunc. Nam et lacus neque.</p>

                            </div>
                            <!-- /ITEM -->

                            <!-- ITEM -->
                            <div class="timeline-item timeline-item-bordered">
                                <!-- timeline entry -->
                                <div class="timeline-entry rounded"><!-- .rounded = entry -->
                                    06<span>offer</span>
                                    <div class="timeline-vline"><!-- vertical line --></div>
                                </div>
                                <!-- /timeline entry -->

                                <h2 class="uppercase bold size-17">We partner with you to keep you informed and engaged.</h2>
                                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas metus nulla, commodo a sodales sed, dignissim pretium nunc. Nam et lacus neque. Ut enim massa, sodales tempor convallis et, iaculis ac massa. Maecenas metus nulla, commodo a sodales sed, dignissim pretium nunc. Nam et lacus neque.</p>

                            </div>
                            <!-- /ITEM -->

                            <!-- ITEM -->
                            <div class="timeline-item timeline-item-bordered">
                                <!-- timeline entry -->
                                <div class="timeline-entry rounded"><!-- .rounded = entry -->
                                    07<span>offer</span>
                                    <div class="timeline-vline"><!-- vertical line --></div>
                                </div>
                                <!-- /timeline entry -->

                                <h2 class="uppercase bold size-17">We deliver your solution on time and on budget.</h2>
                                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas metus nulla, commodo a sodales sed, dignissim pretium nunc. Nam et lacus neque. Ut enim massa, sodales tempor convallis et, iaculis ac massa. Maecenas metus nulla, commodo a sodales sed, dignissim pretium nunc. Nam et lacus neque.</p>

                            </div>
                            <!-- /ITEM -->

                            <!-- ITEM -->
                            <div class="timeline-item timeline-item-bordered">
                                <!-- timeline entry -->
                                <div class="timeline-entry rounded"><!-- .rounded = entry -->
                                    08<span>offer</span>
                                    <div class="timeline-vline"><!-- vertical line --></div>
                                </div>
                                <!-- /timeline entry -->

                                <h2 class="uppercase bold size-17">We are here for you beyond “go live.”</h2>
                                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas metus nulla, commodo a sodales sed, dignissim pretium nunc. Nam et lacus neque. Ut enim massa, sodales tempor convallis et, iaculis ac massa. Maecenas metus nulla, commodo a sodales sed, dignissim pretium nunc. Nam et lacus neque.</p>

                            </div>
                            <!-- /ITEM -->

						
						</div>

					</div>

				</div>
			</section>
			<!-- / -->

            <!-- -->
            <section>
                <div class="container">

                    <div class="margin-top-10">
                        <ul class="nav nav-tabs nav-clean">
                            <li class="active"><a href="#tab1" data-toggle="tab">What We Do For Startup Businesses</a></li>
                            <li><a href="#tab2" data-toggle="tab">What We Do For Mid-Market Businesses</a></li>
                            <li><a href="#tab3" data-toggle="tab">What We Do For Enterprise Businesses</a></li>
                        </ul>

                        <div class="tab-content">
                            <div id="tab1" class="tab-pane fade in active">
                                <img class="pull-left" src="assets/images/demo/mockups/600x399/11-min.jpg" width="200" alt="" />
                                <p>So you have a startup business that is growing fast. You are nimble, smart and self-sufficient. Unfortunately, you don't have time to set up a workstation and access privileges to gain fast, efficient results for your growing startup without a big financial capacity.
                                    <a href="page-business-small">
                                        Read
                                        <!-- /word rotator -->
									<span class="word-rotator" data-delay="2000">
										<span class="items">
											<span>more</span>
											<span>now</span>
										</span>
									</span><!-- /word rotator -->
                                        <i class="glyphicon glyphicon-menu-right size-12"></i>
                                    </a>
                                <p>
                            </div>
                            <div id="tab2" class="tab-pane fade out ">
                                <img class="pull-left" src="assets/images/demo/mockups/600x399/12-min.jpg" width="200" alt="" />
                                <p>So, you have successfully made it through the many challenges of setting and running a startup business. Now, you have a mid market business: you are better than when you started. Unfortunately, growing challenges can happen to any business, and these setbacks can stop you from achieving your ultimate goals, right! Well, we are experienced, dependable, using advanced technology solutions that match your unique need at all stages.
                                    <a href="page-business-mid">
                                        Read
                                        <!-- /word rotator -->
									<span class="word-rotator" data-delay="2000">
										<span class="items">
											<span>more</span>
											<span>now</span>
										</span>
									</span><!-- /word rotator -->
                                        <i class="glyphicon glyphicon-menu-right size-12"></i>
                                    </a>
                                <p>
                            </div>
                            <div id="tab3" class="tab-pane fade">
                                <img class="pull-left" src="assets/images/demo/mockups/600x399/18-min.jpg" width="200" alt="" />
                                <p>We understand that operating an enterprise is not walk in the park. The seemingly challenging task of embracing innovative technologies and incorporating the daily activities with conventional settings need to be addressed. Your enterprise must be able to maneuver in different contexts and keep developing the competencies to keep up with the accelerating pace of the changing business environment. We are experienced, dependable, using innovative technology solution that matches your unique need at all stages of the development processes for your enterprise.
                                    <a href="page-business-large">
                                        Read
                                        <!-- /word rotator -->
									<span class="word-rotator" data-delay="2000">
										<span class="items">
											<span>more</span>
											<span>now</span>
										</span>
									</span><!-- /word rotator -->
                                        <i class="glyphicon glyphicon-menu-right size-12"></i>
                                    </a>
                                <p>
                            </div>
                        </div>
                    </div>

                </div>
            </section>

			<!-- BUTTON CALLOUT -->
			<a href="page-contact" class="btn btn-xlg btn-primary size-20 fullwidth nomargin noradius padding-40">
				<span class="font-lato size-30">
					Are you convinced ?
					<strong>Contact us &raquo;</strong>
				</span>
			</a>
			<!-- /BUTTON CALLOUT -->


			<!-- FOOTER -->
            <?php require_once("footer.html")?>
			<!-- /FOOTER -->

		</div>
		<!-- /wrapper -->


		<!-- SCROLL TO TOP -->
		<a href="#" id="toTop"></a>


		<!-- PRELOADER -->
		<div id="preloader">
			<div class="inner">
				<span class="loader"></span>
			</div>
		</div><!-- /PRELOADER -->


		<!-- JAVASCRIPT FILES -->
		<script type="text/javascript">var plugin_path = 'assets/plugins/';</script>
		<script type="text/javascript" src="assets/plugins/jquery/jquery-2.1.4.min.js"></script>

		<script type="text/javascript" src="assets/js/scripts.js"></script>
		


	</body>
</html>